cdc/owner/changefeed.go

```diff
@@ -437,6 +446,8 @@ func (c *changefeed) releaseResources(ctx cdcContext.Context) {
 	}
 	c.wg.Wait()
 	c.scheduler.Close(ctx)
+	c.scheduler = nil
+	c.barriers = nil
```

Comment:
Please add comment on the life cycle of `barriers` at the definition of `c.barriers`.